.gitignore

```diff
@@ -66,10 +66,7 @@ target/
 Cargo.lock
 **/*.rs.bk
 
-x/programs/simulator/libsimulator.h
-x/programs/runtime/debug
-x/programs/runtime/wasm32-unknown-unknown
-x/programs/runtime/.rustc_info.json
+x/programs/runtime/target
```

Comment:
Should this be updated to `x/contracts` ?

Reply:
good catch